CONTRIBUTING.md

```diff
@@ -1,45 +1,113 @@
-# How to contribute
+# Welcome
 
-Kubeedge is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into kubeedge.
+Welcome to KubeEdge!
 
-# Email and chat
+-   [Before you get started](#before-you-get-started)
+    -   [Code of Conduct](#code-of-conduct)
+    -   [Community Expectations](#community-expectations)
+-   [Getting started](#getting-started)
+-   [Your First Contribution](#your-first-contribution)
+    -   [Find something to work on](#find-something-to-work-on)
+        -   [Find a good first topic](#find-a-good-first-topic)
+        -   [Work on an Issue](#work-on-an-issue)
+        -   [File an Issue](#file-an-issue)
+-   [Contributor Workflow](#contributor-workflow)
+    -   [Creating Pull Requests](#creating-pull-requests)
+    -   [Code Review](#code-review)
+    -   [Testing](#testing)
 
-- Email: [kubeedge](https://groups.google.com/forum/?hl=en#!forum/kubeedge)
-- Slack: [kubeedge](https://join.slack.com/t/kubeedge/shared_invite/enQtNDg1MjAwMDI0MTgyLTQ1NzliNzYwNWU5MWYxOTdmNDZjZjI2YWE2NDRlYjdiZGYxZGUwYzkzZWI2NGZjZWRkZDVlZDQwZWI0MzM1Yzc)
+# Before you get started
 
-## Getting started
+## Code of Conduct
+
+Please make sure to read and observe our [Code of Conduct](https://github.com/kubeedge/kubeedge/blob/master/CODE_OF_CONDUCT.md).
+
+## Community Expectations
+
+KubeEdge is a community project driven by its community which strives to promote a healthy, friendly and productive environment.
+The goal of the community is to develop a cloud native edge computing platform which takes advantages of the features provided by a popular orchestration technology like Kubernetes to manage edge nodes and devices at scale and demonstrate resiliency, reliability in offline scenarios. To build a platform at such scale requires the support of a community with similar aspirations.
```

Comment:
How about `develop a cloud native edge computing platform built upon kubernetes which takes...`, not like Kubernetes

Reply:
fixed. thanks.